README.md

```diff
@@ -363,6 +362,12 @@ turing-test-spark-operator-webhook-init-ph8ds   0/1     Completed   0          3
 turing-test-turing-init-init-pknvb              0/1     Completed   0          3m44s
 ```
 
+Programatically, you could check if the init job has run successfully with the following command:
+
+```bash
+kubectl wait -n infrastructure --for=condition=complete --timeout=10m job/turing-init-init
+```
```

Comment:
Maybe this can also be added under `Test the Deployment` section in `charts/turing-init/README.md`

Reply:
Ok added.